checker/src/main/java/org/checkerframework/checker/nullness/NullnessValue.java

```diff
@@ -89,9 +89,8 @@ public String toStringSimple() {
                 + ", "
                 + TypesUtils.simpleTypeName(underlyingType)
                 + ", "
-                + (isPolyNullNonNull ? 't' : 'f')
-                + ' '
-                + (isPolyNullNull ? 't' : 'f')
+                + ("poly nn/n = " + (isPolyNullNonNull ? 't' : 'f'))
+                + ("/" + (isPolyNullNull ? 't' : 'f'))
```

Comment:
```suggestion
                + "poly nn/n = "
                + (isPolyNullNonNull ? 't' : 'f')
                + "/"
                + (isPolyNullNull ? 't' : 'f')
```

Reply:
I think I wouldn't parenthesize the sub-expressions. I found it harder to read with the extra parens.
What do you think? I'm not sure my formatting is right, so maybe you need to reformat.

Reply:
Thanks for your suggestion, let me commit this improvemnt, and pull it to have a test.

Reply:
The formatting is correct!